.travis.yml

```diff
@@ -18,12 +18,27 @@ stages:
 
 jobs:
   include:
+    # Test separately: purchase_order_type
     - stage: test
       env:
-        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT="1"
+        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT=1 INCLUDE="purchase_order_type"
     - stage: test
       env:
-        - TESTS=1 ODOO_REPO="OCA/OCB"
+        - TESTS=1 ODOO_REPO="OCA/OCB" INCLUDE="purchase_order_type"
+    # Test separately: purchase_quick
+    - stage: test
+      env:
+        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT=1 INCLUDE="purchase_quick"
+    - stage: test
+      env:
+        - TESTS=1 ODOO_REPO="OCA/OCB" INCLUDE="purchase_quick"
+    # Test all other addons together
+    - stage: test
+      env:
+        - TESTS=1 ODOO_REPO="odoo/odoo" EXCLUDE="purchase_order_type,purchase_quick"
```

Comment:
Hm, it looks like there is a bug in the template: there should be `MAKEPOT=1` here. Fixed in https://github.com/OCA/oca-addons-repo-template/pull/65

Reply:
I made a template release. You can run `copier -f update` to get it.